fxa_auth.py

```diff
@@ -65,10 +65,10 @@ def validate(event, method):
             headers={"content-type": "application/json"})
         try:
             response = request.urlopen(req, timeout=5).read()
-        except error.HTTPError as ex:
+        except Exception as ex:
             raise HandlerException(
                 status_code=ex.code,
-                message = "{} {}".format(ex.msg, ex.fp.read()))
+                message="{} {}".format(ex.msg, ex.fp.read()))
```

Comment:
ex.fp's no longer valid for base Exception

Reply:
thanks. This was a bit of debugging. Adding a general handler for Exceptions for now as well.